docs/dev_guide/contents/tests.rst

```diff
@@ -87,6 +87,15 @@ or::
 
     $ pytest --remote-data=any
 
+
+Mocked remote data tests
+------------------------
+Some of our remote data tests are run using `vcrpy <https://vcrpy.readthedocs.io/en/latest/>`_, which saves a copy of any HTTP requests made within the test so a remote request doesn't have to be run on subsequent test runs.
+To use ``vcrpy`` on a test add the ``@pytest.mark.vcrpy()`` decorator to the test, and **keep the ``@pytest.mark.remote_data`` decorator**.
+These should only be used to test search requests, and **not** data download requests, to avoid storing large files in the git repository.
```

Comment:
How bad is it if we were to try to download the data?

Reply:
The cassette files end up being as large as the downloaded files. So mileage may vary depending on the data being downloaded. Perhaps lets start with non-binary files, and then we can have a discussion about pros/cons of including the binary data?

Reply:
We can punt it to another PR/issue for the future, I just wondered if we could use this for the .

In theory if we never change the times, the data should be same, so I can't imagine it being too much of a problem for a throw away git repo to add it to the history.

Reply:
We should probably avoid this for sanity's sake, we shouldn't need very many tests actually downloading files. We can either pick downloads which are very small for testing fetch or we can make those tests actually online, or we can vcr the query and fetch the files?!

Reply:
Yeah I agree, we would make sure we review our download tests and ensure we arent wasting our time with them.

But the ones that remain, maybe we can try to use vcrpy on them?

Reply:
I am fine with doing that iff we use git-lfs in the cassettes repo.